support-workers/src/test/scala/com/gu/support/workers/lambdas/CreatePaymentMethodSpec.scala

```diff
@@ -70,6 +75,53 @@ class CreatePaymentMethodSpec extends AsyncLambdaSpec with MockContext {
       }
   }
 
+  it should "retrieve a valid DirectDebitPaymentMethod when given valid DD fields" in {
+
+    val createPaymentMethod = new CreatePaymentMethod(mockServices)
+
+    val outStream = new ByteArrayOutputStream()
+
+    createPaymentMethod
+      .handleRequestFuture(wrapFixture(createDirectDebitDigitalPackJson), outStream, context)
+      .map { _ =>
+        // Check the output
+        val createSalesforceContactState = Encoding.in[CreateSalesforceContactState](outStream.toInputStream)
+
+        createSalesforceContactState.isSuccess should be(true)
+        createSalesforceContactState.get._1.paymentMethod match {
+          case dd: DirectDebitPaymentMethod =>
+            withClue("direct debit: " + dd) {
+              dd.PaymentGateway should be(DirectDebitGateway)
+            }
+            dd.Email should be(None)
+          case _ => fail()
```

Comment:
will it report if we do this?
```suggestion
          case _ => fail("Payment method: " + createSalesforceContactState.get._1.paymentMethod)
```
or do we have to do this?
```suggestion
          case _ => withClue("Payment method: " + createSalesforceContactState.get._1.paymentMethod)(fail())
```